src/types/GoodAfterTime.sol

```diff
@@ -67,7 +67,7 @@ contract GoodAfterTime is BaseConditionalOrder {
 
         // Require that the sell token balance is above the minimum.
         if (!(data.sellToken.balanceOf(owner) >= data.minSellBalance)) {
-            revert IConditionalOrder.OrderNotValid(BALANCE_INSUFFICIENT);
+            revert IConditionalOrder.PollTryNextBlock(BALANCE_INSUFFICIENT);
```

Comment:
Wasn't the old behavior intended? In this contract's `@dev` comments I read:

> This order type allows for placing an order [...]. As [...], this allows for placing multiple orders. To ensure that the order is not filled multiple times, a `minSellBalance` is checked before the order is placed.

My understanding is that trying again should be avoided because, once the order is matched, the balance would go below `minSellBalance` and (I suppose) should never go back up.

Reply:
Great catch! ️🕵️